--- standalone/drivers/isdn/mISDN/octvqe/octvqe_linux.c 2021-04-24 18:52:53.476114491 +0000
+++ standalone/drivers/isdn/mISDN/octvqe/octvqe_linux.c 2021-04-24 18:53:08.372583757 +0000
@@ -540,7 +540,7 @@
                        wake_up_interruptible(&pChan->ReadWaitQueue);
 
                        /* Retrieve CPU time before processing. */
-                       pChan->ulTimestampIn = (unsigned long)rdtsc();
+//                     pChan->ulTimestampIn = (unsigned long)rdtsc();
                }
 
                if (fWakeUpWrite == 1) /* wake_up_interruptible waiting on write */
