--- drivers/isdn/mISDN/octvqe/octvqe_linux.c
+++ drivers/isdn/mISDN/octvqe/octvqe_linux.c
@@ -540,7 +540,7 @@
                        wake_up_interruptible(&pChan->ReadWaitQueue);
 
                        /* Retrieve CPU time before processing. */
-                       pChan->ulTimestampIn = (unsigned long)rdtsc();
+//                     pChan->ulTimestampIn = (unsigned long)rdtsc();
                }

                if (fWakeUpWrite == 1) /* wake_up_interruptible waiting on write */

