--- sip.cpp     2021-04-24 16:45:17.223599013 +0000
+++ sip.cpp     2021-04-24 16:52:04.312502995 +0000
@@ -18,14 +18,6 @@
 #include <sofia-sip/stun_tag.h>
 #include <sofia-sip/su_md5.h>
 
-#ifndef SOFIA_SIP_GCC_4_8_PATCH_APLLIED
-#warning ********************************************************
-#warning Please apply the sofia-sip-gcc-4.8.patch !
-#warning If this issue is already fixed, just remove this check.
-#warning ********************************************************
-#error
-#endif
-
 #undef NUTAG_AUTO100
 
 unsigned char flip[256];

